src/engraving/layout/v0/tlayout.cpp

```diff
@@ -3602,28 +3602,32 @@ void TLayout::layout(NoteDot* item, LayoutContext&)
 
 void TLayout::layout(Ornament* item, LayoutContext& ctx)
 {
+    if (item->showCueNote()) {
```

Comment:
Why do you need this? This function should work regardless of showCueNote(). This check probably needs to be at a higher level